flagsmith-jira-app/src/frontend/components/IssueFeatureTables.tsx

```diff
@@ -148,30 +146,64 @@ const IssueFeatureTable = ({
   const featureId = environmentFeatures[0]!.id;
   const featureName = environmentFeatures[0]!.name;
 
+  const projectUrl = `${FLAGSMITH_APP}/project/${environmentFeatures[0]?.project}`;
+
   /** Read feature state for each environment */
-  const readFeatureState = useCallback(
-    async (): Promise<FeatureState> => ({
-      featureId,
-      environments: await Promise.all(
-        environments.map(async (environment) => ({
-          ...(await readEnvironmentFeatureState({
+  const readFeatureState = useCallback(async (): Promise<FeatureState> => {
+    const featureProjectId = environmentFeatures[0]?.project;
```

Comment:
As this is read twice (also in the URL) it might be worth putting in a variable.